src/hotspot/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{cpp,hpp,c,h}]
```

Comment:
There are no C files in Hotspot, nor should there ever be. (There are a handful of .h files, though, for interop purposes)

Reply:
Yes I saw that there are no C files but added it for symmetry with the H files. I don't think it hurts here but I can also remove it. Do you think it should be removed?

Reply:
I think that is rather up to the Hotspot folks to have an opinion on.